### ISSUE TITLE
zsh check fails on dash

### ISSUE THREAD

**Issue by AGSaidi** [conda/constructor#348]
tried on  Ubuntu 18.04

```
./Miniforge3-4.8.3-4-Linux-aarch64.sh: 404: ./Miniforge3-4.8.3-4-Linux-aarch64.sh: [[: not found
```
The [line](https://github.com/conda/constructor/blob/66fd26702b4a56b3a0027c3cd46fa8464895b075/constructor/header.sh#L541)  in question` if [[ $SHELL = *zsh ]] `appears to be a bash-ism that Ubuntu's dash doesn't like.

A portable way to write this is
```bash
case $SHELL in
    *zsh) $PREFIX/bin/conda init zsh ;;
    *) $PREFIX/bin/conda init ;;
esac
```

**Comment by hmaarrfk** [comment]
Seems like this would get merged relatively easily. Do you want to submit the Pull Request?

**Comment by nerohmot** [comment]
I am doing it right now :yum:

**Comment by nerohmot** [comment]
:angry: had to sign a CLA, did do so, email verification also OK, but I guess it takes a while for the CLA to be moved in place :frowning_face: 
Will try again in a few hours.

**Comment by hmaarrfk** [comment]
>  We then need to manually verify your signature

**Comment by nerohmot** [comment]
Any idea how long that will take ?

**Comment by hmaarrfk** [comment]
Not too sure. But it is a Sunday ;)

**Comment by hmaarrfk** [comment]
you can likely submit tye patch to conda forge, see if they want to jump the gun on the release. they minght since it is affecting the project.

**Comment by mcg1969** [comment]
Fixed by #351
